crates/weaver_forge/src/extensions/util.rs

```diff
@@ -131,6 +132,16 @@ pub fn acronym(acronyms: Vec<String>) -> impl Fn(&str) -> String {
     }
 }
 
+fn to_yaml(value: Value) -> Result<String, minijinja::Error> {
+    let value = serde_json::to_value(value)
+        .map_err(|error| minijinja::Error::new(ErrorKind::BadSerialization, error.to_string()))?;
```

Comment:
This is unnecessary, you can pass a reference to `minijinja::Value` directly to `serde_yaml::to_string` because it implements `serde::Serialize`.